moveit_core/robot_trajectory/include/moveit/robot_trajectory/robot_trajectory.h

```diff
@@ -144,32 +145,57 @@ class RobotTrajectory
     return waypoints_.front();
   }
 
-  const std::deque<double>& getWayPointDurations() const
+  std::deque<rclcpp::Duration> const& getDurationFromPreviousDeque() const
```

Comment:
```suggestion
  const std::deque<rclcpp::Duration>& getDurationFromPreviousDeque() const
```
There's no need to sprinkle in stylistic changes. Lets keep style consistent until we can enforce const style with clang-format-14. Same goes for a few other places in this PR. See https://github.com/ros-planning/moveit2/issues/1431

Reply:
I'll go make sure I clean these out. I have a pattern of doing this without thinking about it because I've been writing all my new code this way recently.